modules/ROOT/pages/oauth/oauth2-provider-documentation-reference.adoc

```diff
@@ -140,7 +140,9 @@ Configuration related to authorization code handling and the authorization endpo
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
-| Login Page | String | Relative file path to the web page for the resource owner to provide it's credentials  | www-static/auth.html | {nbsp}
+| Login Page | String | Relative file path to the web page for the resource owner to provide it's credentials. +
+  Keep in mind that in case the default page is not used, if the referenced login page requires external files, they won't be handled by the oauth provider. +
+  For example: If the login page html references an external .css style file, that file won't be found unless explicitly exposed by any mean other than the oauth provider. Check  xref:http/http-documentation.adoc#loadStaticResource[http:load-static-resource] | www-static/auth.html | {nbsp}
```

Comment:
I think it's better to frame it from the positive side, like: that file will need to be exposed, check out this stuff for doing just that.